packages/block-editor/src/components/off-canvas-editor/block-select-button.js

```diff
@@ -70,6 +70,10 @@ function ListViewBlockSelectButton(
 		  )
 		: __( 'Edit' );
 
+	const hasChevron = [ 'core/navigation-submenu', 'post', 'page' ].includes(
+		blockInformation.name
+	);
```

Comment:
This bakes in knowledge about particular blocks into the component itself. Will that affect reusability of the component?

Reply:
Agreed that it should be passed to the component. :+1: